scripts/setup.py

```diff
@@ -6,12 +6,13 @@
 import shutil
 import argparse
 import re
+import enquiries
 
 PACKAGE_SEPARATOR = "."
 ANDROID_MODULE = "app"  # only app module now
 
 
-class Project:
+class Object:
```

Comment:
can we keep `Project` like before? 🙏 `Object` has general meaning imo

Reply:
It's because we will re-use it at [here](https://github.com/nimblehq/flutter_templates/blob/19091d29f20654a6ff63982c445de24e69a316f1/scripts/setup.py#L391), so the `Project` doesn't make sense anymore 🤔

Reply:
In that case, I think we will create the Project object is better.

Create a Project object now, just makes the duplicated code, IMO 🤔

Reply:
@markgravity we don't often refactor the code, so trying to make it clean as possible and the later devs won't do the extra step to refactor code 🤓

Reply:
@manh-t It's not so good IMO, we think too much about the future for the `Project` class, they just the same now. 
And we're not 100% this `Project` class will have other fields in the future 🤔 

```
class Object:
    def __init__(self):
        pass

class Project:
    def __init__(self):
        pass
```

Reply:
Or we should define these fields into the `Project` class
```
    project.project_path = args.project_path
    project.new_package = args.package_name
    project.new_app_name = args.app_name
    project.new_project_name = args.project_name
    project.app_version = args.app_version
    project.build_number = args.build_number
``` 
it makes more sense to create a `Project` class, but now we use it likes a `Object` only 🤔

Reply:
@markgravity yes, I think we should add these fields to the `Project` class to avoid confusion:
```
    project.project_path = args.project_path
    project.new_package = args.package_name
    project.new_app_name = args.app_name
    project.new_project_name = args.project_name
    project.app_version = args.app_version
    project.build_number = args.build_number
```
And do the same to `JsonSerializable` 🙆‍♂️ 
```
class JsonSerializable:
    def __init__(self, field_rename):
        self.field_rename = field_rename
```

Reply:
@manh-t updated 🙇‍♂️